landbosse/model/CollectionCost.py

```diff
@@ -704,21 +813,40 @@ def calculate_costs(self, calculate_costs_input_dict, calculate_costs_output_dic
         calculate_costs_output_dict['Labor Cost USD without weather delays'] =((calculate_costs_output_dict['Days taken for trenching (labor)'] * calculate_costs_output_dict['Labor cost of trenching per day (usd/day)']) + (calculate_costs_output_dict['Total per diem costs (USD)'] + calculate_costs_output_dict['managament_crew_cost_before_wind_delay']))
         calculate_costs_output_dict['Labor Cost USD with weather delays'] = calculate_costs_output_dict['Labor Cost USD without weather delays'] * calculate_costs_output_dict['wind_multiplier']
 
-        trenching_labor_cost_df = pd.DataFrame([['Labor',calculate_costs_output_dict['Labor Cost USD with weather delays'], 'Collection']],
-                                               columns = ['Type of cost', 'Cost USD', 'Phase of construction'])
+        if calculate_costs_input_dict['turbine_rating_MW'] >= 0.1:
+            trenching_labor_cost_df = pd.DataFrame(
+                [['Labor', calculate_costs_output_dict['Labor Cost USD with weather delays'], 'Collection']],
+                columns=['Type of cost', 'Cost USD', 'Phase of construction'])
+        else:
+            trenching_labor_cost_df = pd.DataFrame(
+                [['Labor', calculate_costs_output_dict['Labor Cost USD with weather delays'], 'Small DW Collection']],
+                columns=['Type of cost', 'Cost USD', 'Phase of construction'])
```

Comment:
Again, I don't think we should create a separate operation in the output data for "Small DW Collection". I think this should be removed. See comment above.

Reply:
@eberlea as discussed above, small DW is ≤ 100 kW turbine ratings. Small DW has operations and their corresponding costs that are different from rsmeans data for > 100 kW turbine ratings.

Reply:
Yes, thanks for confirming that the data are different. However, just because the calculations use different cost data doesn't mean that they should be reported under a separate name in the output dictionary. I think we should be using consistent naming conventions for output regardless of whether the model is generated results for a distributed run. Does that make sense? @akey7 and @parangat94: please review and revise.